TwitchDownloaderCore.Tests/IdParseTests.cs

```diff
@@ -22,13 +22,13 @@ public void CorrectlyParsesVodId(string id, VideoPlatform expectedPlatform)
         }
 
         [Theory]
-        [InlineData("https://kick.com/video/9d02cf06-52e8-4023-abd5-16b21d143867", "9d02cf06-52e8-4023-abd5-16b21d143867", VideoPlatform.Kick)]
-        [InlineData("https://kick.com/video/79200ecf-c46d-4d74-b3ad-d06a7e0e5d8d", "79200ecf-c46d-4d74-b3ad-d06a7e0e5d8d", VideoPlatform.Kick)]
+        [InlineData("https://kick.com/streamer8/videos/9d02cf06-52e8-4023-abd5-16b21d143867", "9d02cf06-52e8-4023-abd5-16b21d143867", VideoPlatform.Kick)]
+        [InlineData("https://kick.com/streamer8/videos/79200ecf-c46d-4d74-b3ad-d06a7e0e5d8d", "79200ecf-c46d-4d74-b3ad-d06a7e0e5d8d", VideoPlatform.Kick)]
         [InlineData("https://www.twitch.tv/videos/41546181", "41546181", VideoPlatform.Twitch)] // Oldest VODs - 8
         [InlineData("https://www.twitch.tv/videos/982306410", "982306410", VideoPlatform.Twitch)] // Old VODs - 9
         [InlineData("https://www.twitch.tv/videos/6834869128", "6834869128", VideoPlatform.Twitch)] // Current VODs - 10
         [InlineData("https://www.twitch.tv/videos/11987163407", "11987163407", VideoPlatform.Twitch)] // Future VODs - 11
-        [InlineData("https://www.twitch.tv/kitboga/video/2865132173", "2865132173", VideoPlatform.Twitch)] // Alternate highlight URL
+        [InlineData("https://www.twitch.tv/kitboga/streamer8/videos/2865132173", "2865132173", VideoPlatform.Twitch)] // Alternate highlight URL
```

Comment:
You accidentally changed a Twitch URL

Reply:
Ah, I apologise must’ve missed that one. Wasn’t careful with my find and replace  fixed one but must’ve missed that one😅